Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs

```diff
@@ -336,16 +336,19 @@ public IEnumerator ManipulationHandlerOneHandMoveNear()
                 yield return hand.MoveTo(initialGrabPosition, numHandSteps);
                 yield return hand.SetGesture(ArticulatedHandPose.GestureId.Pinch);
 
+                yield return new WaitForFixedUpdate();
```

Comment:
interesting, any idea why we need to add this?

Reply:
Oh, this hung around from when I was trying something. I've tried removing this and the tests still pass.